ingestion/src/metadata/ingestion/source/dashboard/powerbi/metadata.py

```diff
@@ -380,6 +380,35 @@ def yield_bulk_datamodel(
                 )
             )
 
+    def _get_child_measures(self, table: PowerBiTable) -> List[Column]:
+        """
+        Extract the measures of the table
+        """
+        measures = []
+        for measure in table.measures or []:
+            try:
+                measure_type = (
+                    DataType.MEASURE_HIDDEN
+                    if measure.isHidden
+                    else DataType.MEASURE_VISIBLE
+                )
+                description_text = (
+                    f"{measure.description}\n\nExpression : {measure.expression}"
+                    if measure.description
+                    else f"Expression : {measure.expression}"
+                )
+                parsed_measure = {
+                    "dataType": measure_type,
+                    "dataTypeDisplay": measure_type,
+                    "name": measure.name,
+                    "description": description_text,
+                }
```

Comment:
instead of dictionary, let's use a basemodel here
it improves in structure,validation, and maintainability